### PR TITLE
Update dependent modules (bump versions)

### LINKED ISSUE #34: Update dependent modules
## What is the issue?
- Update minio ~> 0.2.0
- Update postgres ~> 0.2.0
## Suggestion(s)/solution(s) [Optional]
Blockers 
- [milestone postgres](https://github.com/fredrikhgrelland/terraform-nomad-postgres/milestone/3)
- [milestone minio](https://github.com/fredrikhgrelland/terraform-nomad-minio/milestone/5)
## Checklist (after created issue)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### PR DESCRIPTION
**Pull request by zhenik**
## Closes/fixes/resolves issue(s)?
- Closes #34 

## What was added/changed/fixed?
- [added] volumes nomad policies 
- [added] create directories  for volumes
- [update] doc, example

## Checklist (after created PR)
- [x] Added reviewers
- [X] Added assignee(s)
- [X] Added label(s)
- [X] Added to project
- [X] Added to milestone

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -2,5 +2,12 @@
 
 All examples have their own directories, with a `main.tf` that references one or more modules. See [template_example](../template_example/example/standalone).
```
Comment (claesgill) [inline example/README.md:3]:
```suggestion
All examples have their own directories, with a `main.tf` that references one or more modules.
```

File: example/main.tf [hunk 2]
```diff
@@ -12,21 +13,23 @@ module "minio" {
   container_image                 = "minio/minio:latest" # todo: avoid using tag latest in future releases
   access_key                      = "minio"
   secret_key                      = "minio123"
```
Comment (claesgill) [inline example/main.tf:15]: Think these can be removed if you want to.

File: example/main.tf [hunk 3]
```diff
@@ -35,7 +38,9 @@ module "postgres" {
   admin_user                      = "hive"
   admin_password                  = "hive"
```
Comment (claesgill) [inline example/main.tf:39]: Same here

**Review by dangernil — APPROVED** [review]

File: example/README.md (hunk 1 above)
Comment (claesgill) [inline example/README.md:3]: This section should be updated with info of the example provided.

Also I think that the example files should go in it's own folder inside the examples folder. Some thing like:
```sh
example/
 ├── standalone_hive
 └── README.md
```

File: example/main.tf (hunk 2 above)
Comment (zhenik) [reply example/main.tf:15]: Can I use my own credentials instead? I want to focus on hive module example

Comment (claesgill) [reply example/main.tf:15]: I don't mind. Let them be as is if you like :+1:

**Comment by claesgill** [comment]
Should we also include the same tests(playbooks) as in postgres and minio? :thinking:

**Comment by zhenik** [comment]
> Should we also include the same tests(playbooks) as in postgres and minio? 🤔

Do you mean https://github.com/fredrikhgrelland/terraform-nomad-hive/blob/master/dev/ansible/03_healthcheck_test.yml ?

**Comment by claesgill** [comment]
> > Should we also include the same tests(playbooks) as in postgres and minio? thinking
> 
> Do you mean https://github.com/fredrikhgrelland/terraform-nomad-hive/blob/master/dev/ansible/03_healthcheck_test.yml ?

I was actually thinking about the volume stests like postgres [04_test_persitence.yml](https://github.com/fredrikhgrelland/terraform-nomad-postgres/blob/master/dev/ansible/04_test_persistence.yml), but healthchecks should also be included in that case. Or what do you think?

**Comment by zhenik** [comment]
> > > Should we also include the same tests(playbooks) as in postgres and minio? thinking
> > 
> > 
> > Do you mean https://github.com/fredrikhgrelland/terraform-nomad-hive/blob/master/dev/ansible/03_healthcheck_test.yml ?
> 
> I was actually thinking about the volume tests, like postgres [04_test_persitence.yml](https://github.com/fredrikhgrelland/terraform-nomad-postgres/blob/master/dev/ansible/04_test_persistence.yml), but healthchecks should also be included in that case. Or what do you think?

Hive is stateless. I see the value of testing persistence only for services that are responsible for persistent data.

**Comment by claesgill** [comment]
> > > > Should we also include the same tests(playbooks) as in postgres and minio? thinking
> > > 
> > > 
> > > Do you mean https://github.com/fredrikhgrelland/terraform-nomad-hive/blob/master/dev/ansible/03_healthcheck_test.yml ?
> > 
> > 
> > I was actually thinking about the volume tests, like postgres [04_test_persitence.yml](https://github.com/fredrikhgrelland/terraform-nomad-postgres/blob/master/dev/ansible/04_test_persistence.yml), but healthchecks should also be included in that case. Or what do you think?
> 
> Hive is stateless. I see the value of testing persistence only for services that are responsible for persistent data.

That's a valid point, and I agree with that.

**Review by claesgill — APPROVED** [review]
LGTM 🚀

**Review by dangernil — APPROVED** [review]

**Pull request merged**